CHANGELOG.md

```diff
@@ -3,6 +3,84 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.14
+
+_Jan 5, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 📆 Add `SingleInputTimeRangeField` and `SingleInputDateTimeRangeField` components
+- 🚀 Use grid for modifying pickers layout
```

Comment:
@mui/xgrid Do we have any other bigger changes worth highlighting?

Reply:
#7272 is a big change, but people not already using the fields probably don't care

Reply:
I've added it as well as added mentions and PR refs to those significant changes.